frontend/src/components/maps/Map.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { GoogleMap, useJsApiLoader} from "@react-google-maps/api";
+import { containerStyle, center, options } from "./mapSettings";
+import {fetchAllKiosks} from "../../service/apiService";
+import {Kiosk} from "../../service/models";
+import '../Components.css';
+
+const Map: React.FC = () => {
+
+    const { isLoaded } = useJsApiLoader({
+        id: 'google-map-script',
+        googleMapsApiKey: process.env.REACT_APP_GOOGLE_API_KEY!
+    })
+
+    const mapRef = React.useRef<google.maps.Map|null>(null);
+
+    const onLoad = (map: google.maps.Map): void => {
```

Comment:
Für deine spätere Appfunktionalität wäre es bestimmt gut, die Kioske dann doch zu speichern und nicht hier nur einmal auf die Karte zu setzen. Vermutlich sogar in einer Parentcomponente, damit zb eine Kioskdetails page laden kann und wenn man zurück kommt nicht wieder alle neu geladen werden müssen. 
Aber nur für die Zukunft